website/webpack.config.js

```diff
@@ -190,12 +190,6 @@ module.exports = env => {
       logInstruction('You can get the token at https://www.dropbox.com/developers');
       throw new Error('Missing Export DropboxClientId Access token');
     }
-    if (!process.env.MapboxExportToken) {
```

Comment:
why remove this? The process should exit if toke nare missing

Reply:
this is actually only optional now. We have decided to not provide a default token until we have more info from Mapbox.

Reply:
gonna undo based on this: https://github.com/uber/kepler.gl/issues/446